index.html

```diff
@@ -0,0 +1,2199 @@
+              </p>
+            </div>
+          </div>
+        </article>
+        <article class="more-stories-article d-flex border-bottom">
+          <div class=" justify-content-end">
+            <img class="img-fluid more-stories_img" src="https://d.newsweek.com/en/full/1581093/coronavirus-america-recoveries-40000-people.jpg" alt="more stories">
+          </div>
+          <div class="d-flex flex-column ml-3">
+            <div>
+              <a href="#">
+                <h4 class=" more-stories-title">
+                  Restauranteur Delivers Oranges to Hospital Workers Amid Coronavirus Crisis
+                </h4>
+              </a>
+            </div>  
+            <div>
+              <p class=" more-stories-paragraph text-body">
+                "We all have to help each other," Matthew Fermin, 
+                owner of East Harlem restaurant Santiago's 
+                Beer Garden, told Newsweek.
+              </p>
+            </div>
+          </div>
+        </article>
+        <article class="more-stories-article d-flex border-bottom">
+          <div class=" justify-content-end">
+            <img class="img-fluid more-stories_img" src="https://d.newsweek.com/en/full/1581093/coronavirus-america-recoveries-40000-people.jpg" alt="more stories">
+          </div>
+          <div class="d-flex flex-column ml-3">
+            <div>
+              <a href="#">
+                <h4 class=" more-stories-title">
+                  Restauranteur Delivers Oranges to Hospital Workers Amid Coronavirus Crisis
+                </h4>
+              </a>
+            </div>  
+            <div>
+              <p class=" more-stories-paragraph text-body">
+                "We all have to help each other," Matthew Fermin, 
+                owner of East Harlem restaurant Santiago's 
+                Beer Garden, told Newsweek.
+              </p>
+            </div>
+          </div>
+        </article>
+        <article class="more-stories-article d-flex">
+          <div class=" justify-content-end">
+            <img class="img-fluid more-stories_img" src="https://d.newsweek.com/en/full/1581093/coronavirus-america-recoveries-40000-people.jpg" alt="more stories">
+          </div>
+          <div class="d-flex flex-column ml-3">
+            <div>
+              <a href="#">
+                <h4 class=" more-stories-title">
+                  Restauranteur Delivers Oranges to Hospital Workers Amid Coronavirus Crisis
+                </h4>
+              </a>
+            </div>  
+            <div>
+              <p class=" more-stories-paragraph text-body">
+                "We all have to help each other," Matthew Fermin, 
+                owner of East Harlem restaurant Santiago's 
+                Beer Garden, told Newsweek.
+              </p>
+            </div>
+          </div>
+        </article>
+        
+
+      </section>
+
+      <!-- first section on large screen -->
+
+      <div class="row first-section-wrapper">
```

Comment:
You shouldn't do this either. The good practice of responsive pages is not creating duplicated content, but playing with classes to re-arrange, hide, and show the elements you want. Please delete the duplicated code and use bootstrap to arrange the elements like the original page.